### PR TITLE
Inventions indicator and schema

### LINKED ISSUE #73: Number of patents (PATSTAT)
**Summary**
- Date: 
- Sources: PATSTAT

**Progress**
- [x] Scoped
- [x] Collected
- [x] Calculated
- [x] Uploaded
- [ ] Raw Schema
- [ ] Processed
- [ ] Processed Schema

**Challenges and Observations**

### PR DESCRIPTION
**Pull request by Juan-Mateos**
Closes #73

### CODE REVIEW

File: data/processed/patents/total_inventions.csv [hunk 1]
```diff
@@ -0,0 +1,201 @@
+,year,nuts_id,nuts_year_spec,total_inventions
```
Comment (mindrones) [inline data/processed/patents/total_inventions.csv]: please remove the index column here

File: data/processed/patents/total_inventions.csv [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+0,2013,UKC1,2013,157
```
Comment (mindrones) [inline data/processed/patents/total_inventions.csv]: Have you run autonuts here? I'm not sure if `year` being same as `nuts_year_spec` would be correct

File: data/processed/patents/total_inventions.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+api: mySQL
```
Comment (mindrones) [inline data/processed/patents/total_inventions.yaml]: This needs to be update to the newer schema template

**Review by mindrones — CHANGES_REQUESTED** [review]
some change required

**Comment by Juan-Mateos** [comment]
Changed schema and updated `nuts_year_spec`

File: data/processed/patents/total_inventions.csv (hunk 2 above)
Comment (mindrones) [reply data/processed/patents/total_inventions.csv]: Years now seem correct but we need to fix the first column

File: data/processed/patents/total_inventions.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    id: total_inventions
+    label: Total unique inventions
+    data_type: int
+  year:
+    data_type: int
+    label: Year
+source_name: PATSTAT (https://www.epo.org/searching-for-patents/business/patstat.html)
+year_range: [2013, 2018]
+
+# [1] https://docs.python.org/3/library/string.html#format-specification-mini-language
```
Comment (mindrones) [inline data/processed/patents/total_inventions.yaml]: minor, I'd remove these 2 lines

**Review by mindrones — CHANGES_REQUESTED** [review]
still some change required

File: data/processed/patents/total_inventions.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+framework_group: knowledge_exchange
+is_experimental: True
+order: [year, nuts_id, nuts_year_spec, value.id]
+region:
+  type: NutsRegion
+  level: 2
+  source: provider
+schema:
+  nuts_id:
+    type: GeoRegion.nuts_id
```
Comment (mindrones) [inline data/processed/patents/total_inventions.yaml]: This should be `NutsRegion.nuts_id`

File: data/processed/patents/total_inventions.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  nuts_year_spec:
+    type: GeoRegion.nuts_year_spec
```
Comment (mindrones) [inline data/processed/patents/total_inventions.yaml]: This should be `type: NutsRegion.nuts_year_spec`

File: data/processed/patents/total_inventions.yaml (hunk 4 above)
Comment (mindrones) [reply data/processed/patents/total_inventions.yaml]: minor, but these are still in

**Review by mindrones — APPROVED** [review]
Good to merge once addressed the suggested changes

**Comment by Juan-Mateos** [comment]
Fixed schema

**Review by mindrones — APPROVED** [review]
Good to merge.

**Pull request merged**